CMake/cdat_modules_extra/git_clone.sh.in

```diff
@@ -3,7 +3,7 @@
 cd "@CMAKE_INSTALL_PREFIX@"
 "@GIT_EXECUTABLE@" clone --no-checkout --depth 1 -b @BRANCH@ @GIT_URL@ "@GIT_TARGET@"
```

Comment:
On certain git versions it does not (you should see reporting on this if you do a google search). I would suggest, we take out the -b option, and just checkout the right branch / tag afterwards.